testing/mm/mm_main.c

```diff
@@ -113,8 +113,6 @@ static const int g_alignment2[NTEST_ALLOCS / 2] =
 static FAR void       *g_allocs[NTEST_ALLOCS];
 static struct mallinfo g_alloc_info;
 
-static dq_queue_t g_realloc_queue;
```

Comment:
remove `#include <mm.h>` at line 35 and search patch from Make.defs

Reply:
OK, let's do this in next pr.

Reply:
Sure.